Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Publish-UpgradedServiceFabricApplication.ps1

```diff
@@ -399,3 +409,63 @@ function Publish-UpgradedServiceFabricApplication
         }
     }
 }
+
+function Invoke-ActionWithRetriesUtil {
+    [CmdletBinding()]
+    param(
+        [scriptblock]
+        $Action,
+
+        [scriptblock]
+        $ActionSuccessValidator = { $true },
+
+        [int32]
+        $MaxTries = 10,
+
+        [int32]
+        $RetryIntervalInSeconds = 1,
+
+        [string[]]
+        [ValidateScript({[System.Exception].IsAssignableFrom([type]$_)})]
+        $RetryableExceptions,
+
+        [string]
+        $RetryMessage,
+
+        [string]
+        $RefreshToken,
+
+        [object]
+        $ConnectedServiceEndpoint
+    )
+
+    try
+    {
+        $upgradeStatus = Invoke-ActionWithRetries -Action $upgradeStatusFetcher `
+            -ActionSuccessValidator $upgradeStatusValidator `
+            -MaxTries $MaxTries `
+            -RetryIntervalInSeconds $RetryIntervalInSeconds `
+            -RetryableExceptions @("System.Fabric.FabricTransientException") `
+            -RetryMessage (Get-VstsLocString -Key SFSDK_WaitingForUpgrade)
+        return $upgradeStatus
+    }
+    catch
+    {
+        if($_.Exception.ErrorCode -eq "InvalidCredentials")
+        {
+            $clusterConnectionParams = @{}
```

Comment:
Does this empty connection parameter work?

Reply:
Yes. Refer this https://github.com/Microsoft/vsts-tasks/blob/master/Tasks/ServiceFabricDeployV1/deploy.ps1#L39